docs/indexing/bm25.mdx

```diff
@@ -73,15 +73,15 @@ WITH (
       the BM25 score.
     </ParamField>
     <ParamField body="tokenizer" default="default">
-      The name of the tokenizer. See [tokenizers]([indexing/bm25#tokenizers]) for a list of available tokenizers.
+      The name of the tokenizer. See [tokenizers](#tokenizers) for a list of available tokenizers.
```

Comment:
Thanks for flagging this! The fix is slightly incorrect, though. The URLs should be:

```
https://docs.paradedb.com/indexing/bm25#normalizers
```

So we need to remove teh `indexing` but keep the `/bm25` part